mlir/lib/Dialect/SPIRV/Transforms/LowerABIAttributesPass.cpp

```diff
@@ -80,7 +80,8 @@ createGlobalVarForEntryPointArgument(OpBuilder &builder, spirv::FuncOp funcOp,
 /// with an spirv.EntryPointOp. Traverses the body of a entry function to do so.
 static LogicalResult
 getInterfaceVariables(spirv::FuncOp funcOp,
-                      SmallVectorImpl<Attribute> &interfaceVars) {
+                      SmallVectorImpl<Attribute> &interfaceVars,
+                      const spirv::TargetEnv &targetEnv) {
```

Comment:
Can we loo up the target env instead of passing it as a function argument?

Reply:
Done